htdocs/accountancy/bookkeeping/list.php

```diff
@@ -548,7 +548,9 @@
 // Code journal
 if (! empty($arrayfields['t.code_journal']['checked']))
 {
-	print '<td class="liste_titre center"><input type="text" name="search_ledger_code" size="3" value="' . $search_ledger_code . '"></td>';
+	print '<td class="liste_titre">';
+	print $formaccounting->select_journal($search_ledger_code, 'search_ledger_code', 0, 1, 1, 1);
```

Comment:
Combo list add more ease of use to make a common filter.

But, it will no more be possible to make search like this
"VT"  (to filter on both VT1 and VT2 for example)
or
"VT | BQ"  to get VT + BQ as a filter
of 
"^VT"  (start with VT).

Who prefer combo and who prefer the free text ?